<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportMatricesFromMatlab</class>
 <widget class="QDialog" name="ImportMatricesFromMatlab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImportFieldsFromMatlab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Choose .MAT file to read</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="fileNameLineEdit_">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFileButton_">
        <property name="text">
         <string>Open...</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="portListWidget_">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Port 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 6</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Matlab object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="matlabObjectListWidget_"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
